person_options.rb

```diff
@@ -2,10 +2,27 @@
 require_relative 'teacher'
 
 class PersonOptions
-  attr_accessor :people
+  attr_accessor :people, :people_objects
 
   def initialize
     @people = []
+    @people_objects = []
+  end
+
+  def teacher_to_object(teacher)
+    {
+      name: teacher.name,
+      age: teacher.age,
+      specialization: teacher.specialization
+    }
+  end
+
+  def student_to_object(student)
+    {
+      name: student.name,
+      age: student.age,
+      classroom: student.classroom
```

Comment:
Hello @Metaverse-Nabeel,

Could you add the id in the creation of teacher_to_object and student_to_object?

currently, we are only storing name, age, specialization for teachers and name, age, and classroom for students. Please take a look if there is anything else to add. It should be something like this
  {
      name: teacher.name,
      age: teacher.age,
      specialization: teacher.specialization,
      ID: teacher.id
    }